src/read_write/s2.rs

```diff
@@ -101,3 +103,39 @@ where
         self.writers.get_mut(cell_id).unwrap()
     }
 }
+
+#[allow(dead_code)]
```

Comment:
Shall we add these functions later, once we use them?

Reply:
I'd like to use them in avmg for some checks in point cloud generation reduce step

Reply:
Ok, then we should remove the `dead_code` allowance and add the two methods to the `pub use self::s2::{...}` section in `src/read_write/mod.rs`